libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -122,10 +123,10 @@ export class SearchFieldComponent implements OnInit, OnDestroy {
      * Set size of search input component.
      */
     @Input()
-    get size(): 'cozy' | 'compact' {
+    get contentDensity(): 'cozy' | 'compact' {
```

Comment:
U dont need this, this is already defined in parent class. Parent class also defines a type for `'cozy' | 'compact'`

Reply:
OK. I've updated the code and removed the `get` and `set` of `contentDensity`. So the code is using the parent input. I had to update the template as well to not use the computed `compact` boolean and instead changed it to `contentDensity === 'compact'`.